server/server.go

```diff
@@ -73,9 +75,25 @@ type DatastoreConfig struct {
 	Timeout      time.Duration `default:"5s"`
 }
 
+type SentryConfig struct {
+	Dsn		string	`default:"https://e8ff473e48aa4962b1a518411489ec5d@o114654.ingest.sentry.io/6398442"`
+	Debug	bool	`default:"true"`
+}
+
```

Comment:
I think it's great you extracted the options in a config struct, but it would also make sense to then parse possibly passed env variables defining this. You can use `envconfig`.